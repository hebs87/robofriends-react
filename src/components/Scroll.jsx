const Scroll = ({children}) => {
  return (
    <div style={{overflowY: 'scroll', border: '1px solid black', height: 700}}>
      {children}
    </div>
  )
}

export default Scroll;
