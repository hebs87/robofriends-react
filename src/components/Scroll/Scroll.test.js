import {shallow} from 'enzyme';
import Scroll from "./Scroll";

describe('Scroll component test suite', () => {
  it('expect to render Scroll component', () => {
    // shallow renders just the target component and not any nest components that are rendered within it
    // toMatchSnapshot() takes a snapshot of the component and tests it is rendering as it should - any change to the
    // component will then cause the test to fail, as it will no longer match the snapshot
    expect(shallow(<Scroll children/>)).toMatchSnapshot();
  });
});
