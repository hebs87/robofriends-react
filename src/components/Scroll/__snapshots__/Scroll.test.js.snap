// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll component test suite expect to render Scroll component 1`] = `
<div
  style={
    Object {
      "border": "1px solid black",
      "height": 700,
      "overflowY": "scroll",
    }
  }
>
  <Component />
</div>
`;
